build.gradle

```diff
@@ -12,7 +12,32 @@ subprojects {
     group = "com.netflix.${githubProjectName}" // TEMPLATE: Set to organization of project
 
     repositories {
-        jcenter()
+        File settingsFile = new File(m2SettingsFile.replace('~', System.env.HOME.toString()))
+        if (settingsFile.isFile()) {
+            println "Detected m2 settings file: ${settingsFile}"
```

Comment:
Apparently, the only way (that I know of) is to hard-code the settings (including credentials) into the build file to access custom repository.